packages/material-ui-lab/src/TabPanel/TabPanel.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+export const styles = {
+  /* Styles applied to the root element. */
+  root: {},
+};
+
+const TabPanel = React.forwardRef(function TabPanel(props, ref) {
+  const { activeValue, children, className, classes, value, ...other } = props;
+
+  return (
+    <Box
+      className={clsx(classes.root, className)}
+      hidden={value !== activeValue}
+      p={3}
+      ref={ref}
+      role="tabpanel"
+      {...other}
+    >
+      {value === activeValue && <Typography>{children}</Typography>}
+    </Box>
+  );
+});
+
+TabPanel.propTypes = {
+  activeValue: PropTypes.any.isRequired,
```

Comment:
I assume the TypeScript to PropTypes generation has an issue

Reply:
What's the issue?

Reply:
Maybe he refers to isRequired?

Reply:
> What's the issue?

@eps1lon There is a `TabPanel.d.ts` with descriptions of the props. Should we expect a "this object was automatically generated from the types" header?

Reply:
Please include this information in the future. The review process is increasingly taking more roundtrips due to missing information from you.

Reply:
done